app/forms/schools/year2020_form.rb

```diff
@@ -78,32 +72,34 @@ def remove_participant(index)
       self.participants = get_participants.filter { |participant| participant[:index] != index }
     end
 
-    def opt_out?
-      induction_programme_choice == "design_our_own" || induction_programme_choice == "no_early_career_teachers"
-    end
-
-    def opt_out!
-      school_cohort = SchoolCohort.find_or_initialize_by(school: school, cohort: cohort)
-      school_cohort.induction_programme_choice = induction_programme_choice
-      school_cohort.save!
-    end
-
     def save!
       ActiveRecord::Base.transaction do
         school_cohort = SchoolCohort.find_or_initialize_by(school: school, cohort: cohort)
         school_cohort.induction_programme_choice = "core_induction_programme"
         school_cohort.core_induction_programme = core_induction_programme
         school_cohort.save!
 
-        get_participants.each do |participant|
+        participants = get_participants.each do |participant|
           EarlyCareerTeachers::Create.call(
             full_name: participant[:full_name],
             email: participant[:email],
             school_cohort: school_cohort,
             mentor_profile_id: nil,
           )
         end
+
+        SchoolMailer.year2020_add_participants_confirmation(
+          user: current_user,
```

Comment:
Can `current_user` be nil? Or did we decide that we will require induction coordinators for NQT+1s?

Reply:
it can't because then we wouldn't know who to send the confirmation email to... presuming we want to send a confirmation email to everyone

Reply:
Really what I am asking is "can this page be accessed without logging in?"

And if it can, well, `current_user` can be nil.

If that happens, surely we can send the email confirmation to the school contact email, right?

Reply:
I've changed it back to the user accessing the page without logging in.